spec/integration/params/multiple_parents_schema_spec.rb

```diff
@@ -10,17 +10,29 @@
   let(:parent1) do
     Dry::Schema.Params do
       required(:email).filled(:string)
+
+      after(:rule_applier) do |result|
+        result.to_h[:foo] = 'foo'
+      end
     end
   end
 
   let(:parent2) do
     Dry::Schema.Params do
       required(:age).filter(:filled?).value(:integer)
+
+      after(:rule_applier) do |result|
+        result.to_h[:bar] = 'bar'
+      end
     end
   end
 
   it 'inherits rules' do
     expect(form.(name: '', age: '').errors.to_h)
       .to eql(email: ['is missing'], age: ['must be filled'], name: ['must be filled'])
   end
+
+  it 'inherits callbacks' do
+    expect(form.(name: '').to_h).to eql(name: '', foo: 'foo', bar: 'bar')
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer the use of `lambda.call(...)` over `lambda.(...)`.](https://app.codacy.com/gh/dry-rb/dry-schema/pullRequest?prid=4764392)